### ISSUE TITLE
we should disable \pagestyle{empty}

### ISSUE THREAD

**Issue by kmccurley** [IACR/latex#191]
I found one paper where the author used `\pagestyle{empty}` and disabled the headers that we set. After we call `\pagestyle{fancy}` we should use
```
\renewcommand{\pagestyle}{\ClassError{iacrcc}{Authors should not change the page style}}
```

**Comment by jwbos** [comment]
Yes, I agree we should try and detect such style changes. Let's investigate if this has any unwanted side-effects.

**Comment by kmccurley** [comment]
The `\pagestyle` command is pretty well documented in the LaTeX user's guide. Once we call `\pagestyle{fancy}` we should be able to disable calls to `\pagestyle`. There is also the `\thispagestyle` command which applies only to the page you're on. `fancyhdr` is merely one way to access these (along with `\markboth`). There are a lot of packages in texlive that use `\pagestyle`, but they all seem to be related to defining their own styles (e.g., exams or beamer). I can find no harm from disabling `\pagestyle` once we call `\pagestyle{fancy}`.

The reason for disabling things like `\pagestyle` is to overcome the extremely bad habits that authors have developed to "just copy what I used before in llncs". People have exercised all sorts of tricks to fit inside 30 pages, including hiding headers and expanding the pagesize. `\pagestyle` appears in 85 out of 127 papers accepted for Crypto 2023 (though some of these are caused by papers that are written with things like `\iflncs`). I think we should definitely disable `\pagestyle`.

**Comment by jwbos** [comment]
Fixed in eefc008cc5d2ca77773f0d7105eb20ef77da5a1c
